### PR TITLE
feat: implement labwc #865 and #1363

### PR DESCRIPTION
**Pull request by ludg1e**
*Apologies if I've disturbed anyone's sleep*

Implements:
- https://github.com/labwc/labwc/issues/865: modifies the text at getting-started.md clarifying that the default files are at /usr/share/doc/labwc and that the users can copy them and paste at .config or modify them
- [x] Should I use "default files" or "example files" wording? Though openbox uses "default" (http://openbox.org/wiki/Help:Configuration)
---
- https://github.com/labwc/labwc/issues/1363: adds a caption to the screenshot at index.md
- [ ] Should the caption have _italic_?
- [ ] Should it be centered?

### CODE REVIEW

**Comment by ludg1e** [comment]
Oh holy, realized the user of #1363 meant the caption at the README rather than labwc.github.io

Anyway, merge or deny/close

**Comment by ahesford** [comment]
I think "default" conveys the wrong impression for the configuration files. They reflect the in-program defaults, but files in that location are never read by the program. "Example" is probably a better term. If there is any ambiguity to resolve, a complex phrase like "Example configuration files, which reflect the default configuration of labwc, [...]" should suffice.

**Comment by Consolatis** [comment]
That panel in the screenshot is not mate-panel but an unreleased one named "Bornel". However we should likely not refer to it as its not publicly available and I don't feel like maintaining it currently. The last time I tested mate-panel it provide a really reduced feature set when running within a wayland session. Maybe we should look into creating a new screenshot with a panel like sfwbar instead.

**Comment by ludg1e** [comment]
I'm waiting for a final consensus about [#7](https://github.com/labwc/labwc-artwork/issues), after that my idea was/is to do some nice wallpapers and push a plain black one with the logo to this PR as the screenshot, with sfwbar

**Comment by Consolatis** [comment]
Good call, I'd like to add #7 to the milestone for 0.7.1 but as its in a different repo, GH won't allow for that.

Personally I still stand by my statement in the thread that the offset to the top should be increased, similar to the orga logo at https://github.com/labwc

**Comment by ludg1e** [comment]
> Personally I still stand by my statement in the thread that the offset to the top should be increased, similar to the orga logo at https://github.com/labwc

I'm not at the day of what is going on in that issue that much

Basically, I'm mostly waiting for that issue could define a final SVG logo that defines the project face.

I also thought like a month ago about suggesting changing the logo font from cantarell (quite deprecated nowadays in my opinion) to something that it's more widely used, though I forgot about it. Maybe now it's a good time to propose it at #7 as well?

**Comment by domo141** [comment]
hmm, failed to notice that "orca" logo even it was mentioned in that # 7...
this is pretty close to that logo:
```
<?xml version="1.0" encoding="UTF-8"?>
<!--labwc logo #2 (C) Johan Malm LICENSE: CC BY-SA 4.0-->
<svg xmlns="http://www.w3.org/2000/svg" width="256px" height="256px"
     viewbox="0 0 256 256"
     stroke-linecap="round" stroke-linejoin="round" stroke-width="12">
  <path fill="#f0d70f" stroke="#f0d70f" d="m 26 68 91 55 v 108 l -76 -61 z" />
  <path fill="#d02f90" stroke="#d02f90" d="m 229 68 -91 55 v 108 l 76 -61 z" />
</svg>
```
(manual trial&errors, fun to a certain point :D)

**Comment by ludg1e** [comment]
Should be ready for review, I believe

Should the caption be centered or have italic?
image

Appreciate feedback regarding the wording of the getting-started page (labwc/labwc #1363):

"""
Example config files, which reflect the default configuration of labwc, are located at `/usr/share/doc/labwc/`. You can either modify those or
create your own ones by copying them to `${XDG_CONFIG_HOME:-$HOME/.config/labwc/}`
(usually `~/.config/labwc/`), with the following five files being used:
[rc.xml], [menu.xml], [autostart], [environment] and [themerc-override].

Your OS/Distribution of choice may include these example configuration files. If not, you could download them with:

```bash
mkdir -p ~/.config/labwc
wget https://raw.githubusercontent.com/labwc/labwc/master/docs/environment -O ~/.config/labwc/environment
wget https://raw.githubusercontent.com/labwc/labwc/master/docs/autostart -O ~/.config/labwc/autostart
wget https://raw.githubusercontent.com/labwc/labwc/master/docs/menu.xml -O ~/.config/labwc/menu.xml
wget https://raw.githubusercontent.com/labwc/labwc/master/docs/rc.xml -O ~/.config/labwc/rc.xml
```
"""

**Review by johanmalm — COMMENTED** [review]
Consider refactoring. It doesn't flow well first saying that config files are located at /usr/share/.... (which they might not be btw) and then say OS/distros might include config files.

File: src/index.md [hunk 1]
```diff
@@ -13,13 +13,14 @@ be controlled with dbus, sway/i3-IPC or other technology. The reason for this is
 that we believe that custom IPCs and protocols create a fragmentation that
 hinders general Wayland adoption.
 
-<a href="img/scrot1.png">
-  <img src="img/scrot1-small.png">
+<a href="img/scrot2.png">
```
Comment (johanmalm) [inline src/index.md:16]: I don't want to change the default screenshot without some dialogue first.

Also, I thing a "figure caption" looks like a report or something. That's why I added a link in template_pre.md instead.

**Comment by johanmalm** [comment]
It's easier to review + merge if you submit changes in smaller batches.
Some of your proposed changes are no-brainers, but others need discussion.
Consider splitting up to separate PRs

File: src/index.md (hunk 1 above)
Comment (ludg1e) [reply src/index.md:16]: There hasn't been added a "figure caption" at any moment

If you refer to this text at top message of the thread, it was just a comment of something I tried and never added (thus why the italic), maybe it was misunderstood
image

Edit: removed

Comment (ludg1e) [reply src/index.md:16]: I suggest removing the caption and when clicking the img (scrot) it redirects you to the obligatory-screenshot page

**Pull request closed without merging**